pkg/cmd/server/api/v1/types_test.go

```diff
@@ -351,7 +353,8 @@ func TestMasterConfig(t *testing.T) {
 		AssetConfig: &internal.AssetConfig{
 			Extensions: []internal.AssetExtensionsConfig{{}},
 		},
-		DNSConfig: &internal.DNSConfig{},
+		DNSConfig:             &internal.DNSConfig{},
+		AdmissionPluginConfig: map[string]string{"pluginName": "configFilePath"}, // Must be specified explicitly becase it's omitempty
```

Comment:
I like that ... having one config file is much better. However, it doesn't play nicely with the current admission control init interface. Not sure how easy it is to get that changed. One option would be to  generate a buffer on the fly with the contents of the config map and pass that to the plugin init function.